doc/index.rst

```diff
@@ -43,99 +87,81 @@ automatic differentiation, and optimization of hybrid quantum-classical computat
 
 ..
 
 - *Best of both worlds*.
-  Support for **hybrid quantum and classical** models
+  Support for **hybrid quantum and classical** models.
 
 ..
 
 - *Batteries included*.
-  Provides **optimization and machine learning** tools
+  Provides **optimization and machine learning** tools.
 
 ..
 
 - *Device independent*.
-  The same quantum circuit model can be **run on different backends**
+  The same quantum circuit model can be **run on different backends**. Install
+  `plugins <https://pennylane.ai/plugins.html>`_ to run your computational circuits
+  on even more devices, including **Strawberry Fields**, **Rigetti Forest**, **ProjectQ**,
+  **Microsoft QDK**, and **IBM Q**.
 
 ..
 
 - *Compatible with existing machine learning libraries*.
   Quantum circuits can be set up to interface with either **NumPy**, **PyTorch**, or **TensorFlow**,
   allowing hybrid CPU-GPU-QPU computations.
 
-..
 
-- *Large plugin ecosystem*.
-  Install plugins to run your computational circuits on more devices, including **Strawberry Fields**, **Rigetti Forest**, **ProjectQ**, **Microsoft QDK**, and **IBM Q**
 
-:html:`<h4>Available plugins</h4>`
+Getting started
+---------------
 
-* `PennyLane-SF <https://github.com/XanaduAI/pennylane-sf>`__: Supports integration with `Strawberry Fields <https://github.com/XanaduAI/strawberryfields>`__, a full-stack Python library for simulating continuous variable (CV) quantum optical circuits.
+For getting started with PennyLane, check out some of the
+`key concepts <https://pennylane.ai/qml/concepts.html>`_ behind quantum machine
+learning, before moving on to some `introductory tutorials <https://pennylane.ai/qml/beginner.html>`_.
 
-..
+Then, take a deeper dive into quantum machine learning by
+exploring cutting-edge algorithms using PennyLane and near-term quantum hardware,
+with our collection of
+`QML tutorials <https://pennylane.ai/qml/implementations.html>`_.
 
-* `PennyLane-Forest <https://github.com/rigetti/pennylane-forest>`_: Supports integration with `PyQuil <https://github.com/rigetti/pyquil>`__, the `Rigetti Forest SDK <https://www.rigetti.com/forest>`__, and the `Rigetti QCS <https://www.rigetti.com/qcs>`__, an open-source quantum computation framework by Rigetti. Provides device support for the Quantum Virtual Machine (QVM) and Quantum Processing Units (QPUs) hardware devices.
+You can also check out our `documentation <https://pennylane.readthedocs.io>`_ for
+more details on the quantum operations, and to explore the available optimization
+tools provided by PennyLane, and detailed guides on
+`how to write your own <https://pennylane.readthedocs.io/en/latest/API/overview.html>`_
+PennyLane-compatible quantum device.
 
-..
+Finally, play around with the numerous `devices and plugins <https://pennylane.ai/plugins.html>`_
+available for running your hybrid optimizations — these include
+IBM Q, provided by the PennyLane-Qiskit plugin, as well as the Rigetti Aspen-1 QPU.
```

Comment:
- provide links to plugins
- mention `pennylane-forest` plugin

Reply:
Done